src/logger.ts

```diff
@@ -65,7 +73,7 @@ export const log: LogCall = (team, ...args) => {
  * Subscribe to a team’s log
  * @param team the team’s unique ID
  */
-const subscribeTo: TeamFunction = (team) => {
+const subscribeTo: TeamFunction = (team: TeamName) => {
```

Comment:
If it’s already part of the `TeamFunction` type, it’s a little redundant.

```suggestion
const subscribeTo: TeamFunction = (team) => {
```